plugins/modules/podman_image.py

```diff
@@ -698,6 +783,9 @@ def build_image(self):
                 f.write(container_file_txt)
             args.extend(['--file', container_file_path])
 
+        if containerfile_hash != "":
```

Comment:
```suggestion
        if containerfile_hash:
```

Reply:
Fixed in https://github.com/containers/ansible-podman-collections/pull/811/commits/ec02a05e52ce73c30c485bd8b726d5aaab0cbe49